react-app/src/components/DB_Trunk.js

```diff
@@ -14,46 +14,35 @@ import DB_Trunk_Cards from "./DB_Trunk_Cards";
 
 import { FaTrash as DeleteIcon } from "react-icons/fa";
 
-function DB_Trunk() {
+function DB_Trunk({ userCards }) {
```

Comment:
Since this was passed as props from the previous component, we take that in here